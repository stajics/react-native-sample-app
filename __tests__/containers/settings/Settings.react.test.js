import 'react-native';
import React from 'react';
import { Settings } from '../../../app/containers/settings/Settings.react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'; // eslint-disable-line

it('renders correctly', () => {
  const settings = renderer.create(
    <Settings
      text={'TEST TEXT'}
      title={'TEST TITLE'}
    />
  ).toJSON();
  expect(settings).toMatchSnapshot();
});
