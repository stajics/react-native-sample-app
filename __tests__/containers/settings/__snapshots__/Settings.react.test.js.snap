exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
      "marginTop": 50,
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    onPress={[Function]}>
    Im the 
    TEST TEXT
     component with title 
    TEST TITLE
  </Text>
</View>
`;
